yeyun/app.js

```diff
@@ -35,6 +35,75 @@ app.get("/ping", (req, res) => {
   res.status(200).json({ message: "pong" });
 });
 
+app.get("/posts", async (req, res) => {
+  const postData = await appDataSource.query(
+    `SELECT
+          users.id as userId,
+          users.profile_iamge as userProfileImage,
+          posts.id as postingId,
+          posts.content as postingContent
+      FROM posts
+      INNER JOIN users ON users.id = posts.user_id;
+      `,
+    (err, rows) => {
+      return res.status(200).json({ data: rows });
+    }
+  );
+});
+
+app.get("/pcheck/:userId/posts", async (req, res) => {
```

Comment:
해당 엔드포인트를 REST하게 다시 작성해 주세요. `/pcheck` 는 어떤 기능을 하는 API인지 유추하기 어렵습니다.
해당 블로그 참고해서 공부하시고 다시 수정해주세요.  [여기](https://velog.io/@somday/RESTful-API-%EC%9D%B4%EB%9E%80)
```suggestion
app.get("/posts/user/:userId", async (req, res) => {
```